kedro-telemetry/README.md

```diff
@@ -20,42 +20,39 @@ for further information on the intent of the data collection and what data is co
+2. **CLI Option When Creating a New Project**:
+   When creating a new project, you can use the command:
 
-When you next run the Kedro CLI you will be asked for consent to share usage analytics data for the purposes explained in the privacy notice, and a `.telemetry` YAML file will be created in the project root directory. The variable `consent` will be set according to your choice in the file, e.g. if you consent:
+   ```console
+   kedro new --telemetry=yes/no
+   ```
+   This will create a `.telemetry` file inside the new project's folder with `consent: true/false` accordingly. This file will be used when executing Kedro commands within that project folder. Note that telemetry data about the execution of the `kedro new` command will still be sent if telemetry has not been disabled using environment variables.
 
-```yaml
-consent: true
-```
+   >*Note:* The `.telemetry` file should not be committed to `git` or packaged in deployment. In `kedro>=0.17.4` the file is git-ignored.
 
->*Note:* The `.telemetry` file should not be committed to `git` or packaged in deployment. In `kedro>=0.17.4` the file is git-ignored.
-
-## How do I withdraw consent?
+3. **Modify or Create the `.telemetry` File**:
+   If the `.telemetry` file exists in the root folder of your Kedro project, set the `consent` variable to `false`. If the file does not exist, create it with the following content:
+     ```yaml
+     consent: false
+     ```
 
-To withdraw consent, you can change the `consent` variable to `false` in `.telemetry` YAML by editing the file in the following way:
+4. **Uninstall the Plugin**:
+   Remove the `kedro-telemetry` plugin:
 
-```yaml
-consent: false
-```
-You can also set `DO_NOT_TRACK` or `KEDRO_DISABLE_TELEMETRY` environment variable to `True`.
-
-Or you can uninstall the plugin:
-
-```console
-pip uninstall kedro-telemetry
-```
+   ```console
+   pip uninstall kedro-telemetry
+   ```
 
 ## What happens when I withdraw consent?
 
-Data will only be collected if consent is given. Otherwise, if consent was explicitly denied or withdrawn, the message below will be printed out on every Kedro CLI invocation. If you explicitly deny consent from the beginning, no data will be collected. If you withdraw consent later, the processing of data will be stopped from that moment on.
+Data collection for telemetry is enabled by default. If consent was explicitly denied or withdrawn, the message below will be printed out on every Kedro CLI invocation. If you explicitly deny consent from the beginning, no data will be collected. If you withdraw consent later, the processing of data will be stopped from that moment on.
```

Comment:
> Data collection for telemetry is enabled by default. 

Maybe we should move it to the `How do I withdraw consent?`. Otherwise, it's a bit confusing that we explain how to withdraw consent at first.

Reply:
I agree with Elena here. It would make more sense to move it to the "How do I withdraw consent?" section.

Reply:
Thanks, agree with you, done.